shell/app/i18n-easy-edit/pages/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import FixedWidget from './fixed-widget';
+import EditModal from './edit-modal';
+import PublishModal from './publish-modal';
+import { getEditCount, isEditAccess } from '../utils';
+
+// insert in root dom
+// erda-ui/shell/app/layout/pages/page-container/components/shell/index.tsx
+const I18nEditPage: React.FC = () => {
+  const [isPublishVisible, setPublishVisible] = React.useState(false);
+  const [editCount, setEditCount] = React.useState(getEditCount());
+  if (!isEditAccess()) return null;
```

Comment:
is it required to check  isEditAccess here, seems it has been checked when init

Reply:
yes, here to check is to determine whether the fixed- widget exists